app/controllers/admin/campaigns_controller.rb

```diff
@@ -4,7 +4,7 @@ class Admin::CampaignsController < AdminController
   # GET /campaigns
   # GET /campaigns.json
   def index
-    @campaigns = Campaign.search(params[:search]).order(sort_column + " " + sort_direction).paginate(:page => params[:page], :per_page => 10)
+    @campaigns = Campaign.search(params[:search]).order(sort_column + " " + sort_direction).paginate(:page => params[:page], :per_page => 9)
```

Comment:
Can you standardise the hash syntax all across? let's use the `{ new: 'syntax' }`